packages/spec/README.md

```diff
@@ -36,12 +24,20 @@ export interface StandardSchema<Input = unknown, Output = Input> {
    */
   readonly "~validate": StandardValidate<Output>;
   /**
-   * The stored type information of the schema.
+   * Inferred types associated with the schema.
    */
-  readonly "~types"?: StandardTypes<Input, Output> | undefined;
+  readonly "~types": StandardTypes<Input, Output> | undefined;
```

Comment:
Did you forget to make it optional again, or is this intentional?

Reply:
I did forget this thank you ☠️